rest/client.go

```diff
@@ -43,14 +45,18 @@ func NewClient(opts ...rest.Option) MeasurementsClient {
 	return &client{Client: restClient}
 }
 
-func (c *client) GetNodeDataRecent(ctx context.Context, nodeID uuid.UUID, contentType []string) (models.ModelNodeDataResponse, error) {
+func (c *client) GetNodeDataRecent(ctx context.Context, nodeID uuid.UUID, contentTypes []string) (models.ModelNodeDataResponse, error) {
 	request := rest.Get("nodes/{nodeID}/node-data/recent{?content_type*}").
 		Assign("nodeID", nodeID.String()).
-		Assign("content_type", contentType).
+		Assign("content_type", workaround.FormatContentTypes(contentTypes)).
 		SetHeader("Accept", "application/json")
```

Comment:
You can avoid your "workaround" by changing,
`{?content_type*}` into `{?content_type}`

https://datatracker.ietf.org/doc/html/rfc6570#section-3.2.8